x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/actions/DownsampleActionIT.java

```diff
@@ -280,6 +287,20 @@ public void testTsdbDataStreams() throws Exception {
         });
     }
 
+    public void testRollupNonTSIndex() throws Exception {
+        createIndex(index, alias, false);
+        index(client(), index, true, null, "@timestamp", "2020-01-01T05:10:00Z", "volume", 11.0, "metricset", randomAlphaOfLength(5));
+
+        String phaseName = randomFrom("warm", "cold");
+        createNewSingletonPolicy(client(), policy, phaseName, new DownsampleAction(ConfigTestHelpers.randomInterval()));
+        updatePolicy(client(), index, policy);
+
+        String rollupIndex = waitAndGetRollupIndexName(client(), index);
+        assertNull("Rollup index should not have been created", rollupIndex);
+        assertBusy(() -> assertTrue("Source index should not have been deleted", indexExists(index)));
+        assertBusy(() -> assertThat(getStepKeyForIndex(client(), index), equalTo(PhaseCompleteStep.finalStep(phaseName).getKey())));
```

Comment:
These assertions make sense, but they are written in a way that would make the test idle for 60 seconds until `waitAndGetRollupIndexName` timeouts.

Would it make sense to have the final step assertion be the first one ?
i.e. 
```
assertBusy(() -> assertThat(getStepKeyForIndex(client(), index), equalTo(PhaseCompleteStep.finalStep(phaseName).getKey())));
```
Followed by regular assertions (they don't need to be `assertBusy` anymore as ILM is in the final step already)
e.g.
```
assertNull("Rollup index should not have been created", rollupIndex);
assertTrue("Source index should not have been deleted", indexExists(index));
```